yt_dlp/extractor/microsoftembed.py

```diff
@@ -0,0 +1,80 @@
+from .common import InfoExtractor
+from ..utils import (
+    traverse_obj,
+    unified_timestamp,
+    int_or_none,
+)
+from collections import OrderedDict
+
+
+class MicrosoftEmbedIE(InfoExtractor):
+    _VALID_URL = r'https?://(?:www\.)?microsoft\.com/en-us/videoplayer/embed/(?P<id>[a-z0-9A-Z]+)'
+
+    _TESTS = [{
+        'url': 'https://www.microsoft.com/en-us/videoplayer/embed/RWL07e',
+        'info_dict': {
+            'id': 'RWL07e',
+            'title': '...'
+        }
+    }]
+    _API_URL = 'https://prod-video-cms-rt-microsoft-com.akamaized.net/vhs/api/videos/'
+
+    def _real_extract(self, url):
+        video_id = self._match_id(url)
+
+        metadata = self._download_json(
+            self._API_URL + video_id,
+            video_id)
```

Comment:
```suggestion
        metadata = self._download_json(self._API_URL + video_id, video_id)
```
Make sure to read our coding conventions